package.json

```diff
@@ -4,14 +4,14 @@
   "description": "Making company travel and accomodation easy and convinient",
   "main": "index.js",
   "scripts": {
-    "test": "npm run migration:refresh:test && NODE_ENV=test npm run db:seed && NODE_ENV=test nyc --reporter=html --reporter=text mocha --timeout 20000 -r @babel/register ./src/modules/**/*.test.js --exit",
+    "test": "npm run migration:refresh:test && npm run db:seed && nyc --reporter=html --reporter=text mocha --timeout 20000 -r @babel/register ./src/modules/**/*.test.js --exit",
     "start": "node dist",
     "build": "rm -rf dist/ && babel src/ -d dist/",
     "start:dev": "nodemon --exec babel-node ./src/index.js",
     "migration": "./node_modules/.bin/sequelize db:migrate",
     "migration:rollback": "./node_modules/.bin/sequelize db:migrate:undo:all",
     "migration:refresh": "npm run migration:rollback && npm run migration",
-    "migration:refresh:test": "NODE_ENV=test npm run migration:rollback && NODE_ENV=test npm run migration",
```

Comment:
Can you please share your thought process for the removal of the NODE_ENV variable?

Reply:
Yes, i removed it because after installing cross-env as a dependency the test didn't run at all,tho it threw no errors but it exited once the script was started.

Reply:
Will look for a way to make it work tho this is just a walk around

Reply:
@orley12, Where did you install `cross-env`. You need to install it as a `dependency` not a `dev-dependency`